src/index.ts

```diff
@@ -0,0 +1,103 @@
+import * as constants  from './constants';
+import dateFormatter  from './utils/date';
+import websocket  from './websocket';
+import * as errors  from './errors';
+import * as ErrorMessages  from './errorMessages';
```

Comment:
We could just import the `ErrorMessages` enum but the wallets use the `hathorLib.ErrorMessages.ErrorMessages...` so I'll keep this "unnecessary encapsulation" so we don't have to refactor the wallets